docs/en/query_language/functions/other_functions.md

```diff
@@ -861,12 +861,69 @@ Takes the states of the aggregate function and returns a column with values, are
 For example, takes state of aggregate function (example runningAccumulate(uniqState(UserID))), and for each row of block, return result of aggregate function on merge of states of all previous rows and current row.
 So, result of function depends on partition of data to blocks and on order of data in block.
 
-## joinGet('join_storage_table_name', 'get_column', join_key) {#other_functions-joinget}
+## joinGet {#joinget}
 
-Gets data from [Join](../../operations/table_engines/join.md) tables using the specified join key.
+The function lets you extract data from the table the same way as from a [dictionary](../dicts/index.md).
+
+Gets data from [Join](../../operations/table_engines/join.md#creating-a-table) tables using the specified join key.
 
 Only supports tables created with the `ENGINE = Join(ANY, LEFT, <join_keys>)` statement.
 
+**Syntax** 
+
+```sql
+joinGet(join_storage_table_name, `value_column`, join_keys)
+```
+
+**Parameters** 
+
+- `join_storage_table_name` — [Identifier](../../syntax/#syntax-identifiers) indicates where search is performed. In most cases, this is the name of the database and/or table. Firstly, identifier is searched in the default database (see parameter `default_database` in the config file), and then in other databases. To override the default database, use the `USE db_name` command.
+    - Specify only the table name for tables with a unique name.
+    - Specify the database and then the table for non-unique tables. Use a dot as the separator.
+- `value_column` — name of the column of the table that contains required data.
+- `join_keys` — list of keys.
+
+**Returned value**
+
+Returns list of values corresponded to list of keys.
+
+If certain doesn't exist in source table then `0` or `null` will be returned based on [join_use_nulls](../../operations/settings/settings.md#join_use_nulls) setting. 
```

Comment:
There's a difficult case there that should be described (in Engine Join and referenced from here). Engine Join allows `join_use_nulls setting` in CREATE TABLE statement. And SELECT query allows `join_use_nulls` too. If you have different `join_use_nulls` you can get an error joining table Engine Join. It depends on kind of JOIN. When you use joinGet() you probably have to use same `join_use_nulls` setting in CRATE TABLE and SELECT statements. It's better check what if not (there was a crash in JOIN in this case).

Reply:
Added it.